paddle/operators/momentum_op.h

```diff
@@ -50,8 +50,7 @@ class MomentumOpKernel : public framework::OpKernel<T> {
 
     v_out.device(place) = v * mu + g;
     if (use_nesterov) {
-      p_out.device(place) = p - g * lr.broadcast(grad_dsize) +
-                            v_out * mu * lr.broadcast(grad_dsize);
+      p_out.device(place) = p - (g - v_out * mu) * lr.broadcast(grad_dsize);
```

Comment:
Whether the `framework::EigenScalar` can be used for `lr` instead of `broadcast` ?

Reply:
I've tried it, but the result is wrong.